src/utilities/algolia.js

```diff
@@ -196,8 +197,7 @@ const stateSettings = {
  * (see https://www.algolia.com/doc/guides/building-search-ui/going-further/backend-search/how-to/highlighting-snippeting/)
  */
 const pageSettings = {
-  attributeForDistinct: 'section',
-  distinct: true,
+  attributeForDistinct: 'contentful_id',
```

Comment:
Finally, it's not resolved: 
* https://www.algolia.com/apps/G1E1Q4HGTB/explorer/browse/live_blogPost
* Search `tracking race`
* We get 3 hits from the same page

The `section` attribute will filter these 3 results based on the auto-incremented value (section0, section1...). 

In fact, you may think the `contentful_id` is used to consolidate the result, like "all results with the same ID should be consolidated" but actually that's the opposite: we _want_ a different attribute for each chunk :)

Reply:
@superbiche - I just re-updated the index, I think a local build had screwed it up:

https://deploy-preview-768--upbeat-lovelace-3e9fff.netlify.app/search?q=tracking%20race

Try "tracking race"

Reply:
Also like Native American, which is in the second chunk for that post, shows up

Reply:
@kevee OK I totally understood the docs in the wrong way. The test index is OK, I guess you can change the live one to use  :+1:

Reply:
Cool, thanks!